xrviz/dashboard.py

```diff
@@ -81,8 +98,25 @@ def create_plot(self, *args):
             if self.var in list(sel_data.coords):  # When a var(coord) is plotted wrt itself
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+            c_low_val = self.kwargs['cmap lower limit']
+            c_upp_val = self.kwargs['cmap upper limit']
+
+            # It is better to set initial values as 0.1,0.9 rather than 0,1 i.e. (min, max)
+            # to get a color balanced graph
+            c_lim_lower = float(c_low_val) if c_low_val else sel_data.quantile(0.1)
+            c_lim_upper = float(c_upp_val) if c_upp_val else sel_data.quantile(0.9)
+
+            color_range = {sel_data.name: (c_lim_lower, c_lim_upper)}
+
+            if not c_low_val:  # if initial values are empty
+                self.control.style.lower_limit.value = str(c_lim_lower.values.round(5))
+                self.control.style.upper_limit.value = str(c_lim_upper.values.round(5))
+
+            if color_scale is not 'linear':
+                sel_data = getattr(numpy, color_scale)(sel_data)  # Color Scaling
+
             assign_opts = {dim: self.data[dim] for dim in sel_data.dims}
-            graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts)
+            graph = sel_data.assign_coords(**assign_opts).hvplot.quadmesh(**graph_opts).redim.range(**color_range)
```

Comment:
I am surprised it is not a `clim=` argument to `quadmesh`. Perhaps `.redim` exists to be able to change the colours without recalculating the grid (but we don't make use of the intermediate object)

Reply:
ping; it would be good to have a comment on the usage here, which seems little odd to me